pkgs/applications/editors/emacs/elisp-packages/manual-packages/cask/package.nix

```diff
@@ -1,67 +1,58 @@
-{ lib
-, stdenv
-, fetchFromGitHub
-, bash
-, emacs
-, python3
+{
+  lib,
+  bash,
+  emacs,
+  fetchFromGitHub,
+  melpaBuild,
+  python3,
 }:
 
-stdenv.mkDerivation (finalAttrs: {
-  pname = "cask";
+let
   version = "0.8.8";
+in
+melpaBuild {
+  pname = "cask";
+  inherit version;
 
   src = fetchFromGitHub {
     owner = "cask";
     repo = "cask";
-    rev = "v${finalAttrs.version}";
+    rev = "v${version}";
     hash = "sha256-TlReq5sLVJj+pXmJSnepKQkNEWVhnh30iq4egM1HJMU=";
   };
 
-  doCheck = true;
+  buildInputs =
+    [
+      bash
+      python3
+    ]
+    ++ (with emacs.pkgs; [
+      ansi
+      dash
+      ecukes
+      el-mock
+      ert-async
+      ert-runner
+      f
+      git
+      noflet
+      package-build
+      s
+      servant
+      shell-split-string
+    ]);
 
-  nativeBuildInputs = [ emacs ];
-  buildInputs = [
-    bash
-    python3
-  ]
-  ++ (with emacs.pkgs; [
-    ansi
-    dash
-    ecukes
-    el-mock
-    ert-async
-    ert-runner
-    f
-    git
-    noflet
-    package-build
-    s
-    servant
-    shell-split-string
-  ]);
+  doCheck = true;
 
   strictDeps = true;
 
-  buildPhase = ''
-    runHook preBuild
-
-    emacs --batch -L . -f batch-byte-compile cask.el cask-cli.el
-
-    runHook postBuild
-  '';
-
-  installPhase = ''
-    runHook preInstall
-
+  postInstall = ''
     mkdir -p $out/bin
-    install -Dm444 -t $out/share/emacs/site-lisp/cask *.el *.elc
-    install -Dm555 -t $out/share/emacs/site-lisp/cask/bin bin/cask
+    install -Dm555 -t $out/share/emacs/site-lisp/cask/bin $sourceRoot/bin/cask
     ln -s $out/share/emacs/site-lisp/cask/bin/cask $out/bin/cask
```

Comment:
`LISPDIR` is changed from `$out/share/emacs/site-lisp/cask` to `$out/share/emacs/site-lisp/elpa/cask-version`.  So these lines need update.